web/profiles/jcc_components_profile/modules/custom/jcc_tc2_all_immutable_config/jcc_tc2_all_immutable_config.install

```diff
@@ -1053,3 +1053,12 @@ function jcc_tc2_all_immutable_config_update_9004() {
     jcc_tc2_all_immutable_config_redirect_extension_strip('jpeg');
   }
 }
+
+/**
+ * Feature import.
+ */
+function jcc_tc2_all_immutable_config_update_9005() {
+  \Drupal::service('features.manager')->import(['jcc_tc2_all_immutable_config']);
```

Comment:
@paboden is this feature import in hook_update similar to what the deployment script does here:
https://github.com/JudicialCouncilOfCalifornia/trialcourt/blob/2c997ff2734598160d2b41ea9fc1cf9e28d3cabb/.circleci/scripts/deploy.sh#L122

> drush @${SITE_CODE}.${PANTHEON_ENV} fra --bundle=jcc_tc2 -y